shuriken/mjolnir/include/passes/mjolnirtoopgraph.h

```diff
@@ -0,0 +1,33 @@
+//--------------------------------------------------------------------*- C++ -*-
+// Shuriken-Analyzer: library for bytecode analysis.
+// @author Farenain <kunai.static.analysis@gmail.com>
+//
+// @file mjolnirtoopgraph.h
+// @brief A Pass to transform a Method from MjolnIR to an operand graph.
+
+#ifndef MJOLNIR_TO_OP_GRAPH_HPP
+#define MJOLNIR_TO_OP_GRAPH_HPP
+
+#include <memory>
+
+#include <mlir/Pass/Pass.h>
+#include <mlir/Support/LLVM.h>
+#include <llvm/Support/raw_ostream.h>
+
+// We need to declare the Pass class
+// from MLIR
+namespace mlir {
+class Pass;
+} // namespace mlir
+
+namespace shuriken::MjolnIR {
```

Comment:
forward declaration from exteral project, i think brings us more trouble that we think 
https://google.github.io/styleguide/cppguide.html#Forward_Declarations